### ISSUE TITLE
Assigning developers to website project

### ISSUE THREAD

**Issue by heinvv** [Forket/connect2sa.co.za_01#8]
Description will be provided later this week

**Comment by Forket** [comment]
Agreed. will assign issue to U in order to wait.

**Comment by heinvv** [comment]
Hi Alex, 

This is a more substantial project that I'd like you to work on. 

Can you please study this document and give us a quote and planning for it?
[20151117 assigning users to projects.docx](https://github.com/Forket/connect2sa.co.za_01/files/37877/20151117.assigning.users.to.projects.docx)

**Comment by heinvv** [comment]
Hi Alex, 

We have updated the document. Can you please use this version (version 2)?
[20151120 assigning users to projects_v2.docx](https://github.com/Forket/connect2sa.co.za_01/files/40028/20151120.assigning.users.to.projects_v2.docx)

**Comment by Forket** [comment]
Thanks for update. 
Today finished training and going back to Poland. And of course as promised all my time is yours. 


Sent from Mobile

Thanks and regards,
Aleksandr Sen
CEO | Expedition PM
Mobile: +48 532 292 764
Skype: aleksandr.sen
Email: alex@sen.pm
Social: LinkedIn | Google+
 
EXPEDITION.PM | KNOWLEDGEMAP.PM

**Comment by Forket** [comment]
>The persons who adds this website, needs to use his/her api key as webmaster verification tool. 

What for user have to use API key? it is NOT needed for creating user.

**Comment by Forket** [comment]
Ready. But will deploy to your env tomorrow

**Comment by heinvv** [comment]
Cool, thanks.

I am not sure about your question about api key.

We use api key in add listing form for websmaster verification.

**Comment by Forket** [comment]
>User recognition

Done

**Comment by heinvv** [comment]
Super cool

I can't check it from here. I only saw your message on the phone.

-----Original Message-----

**Comment by Forket** [comment]
>More advanced

Done with remarks:
-With your current solution it's not possible to use popup for login in process of approving adding to listing (because popup is processed AFTER server processes request)

**Comment by heinvv** [comment]
Is it not possible to save original query or data, and to reload that after finishing the login procedure?

Btw: for which action do we need login?

Or can we check if logged in just before starting the action?



-----Original Message-----

**Comment by Forket** [comment]
>s it not possible to save original query or data, and to reload that after finishing the login procedure?

it already stores. But with redirect.

>Btw: for which action do we need login?

when user which was added to listing approves it from received email.

>Or can we check if logged in just before starting the action?

I do it. but it's ALWAYS on server side.

Summary: now it work and checks and required login if user is not logged in. But without popup. it just redirects user to default login page. In order to change this you have to change login procedure for WHOLE your site and recode general approach which is for now will be expensive. So just try to test. If you want I can record video test for you and you will see.

**Comment by heinvv** [comment]
Great 

I'll test it on Monday

I trust your judgement about this 

-----Original Message-----

**Comment by heinvv** [comment]
I am a bit confused. 
I marked "User recognition" and "More advanced" as not done. 
What server are you working on?

**Comment by Forket** [comment]
[20151207.outstanding.work.from.assigning.users.to.projects_v2.docx](https://github.com/Forket/connect2sa.co.za_01/files/55559/20151207.outstanding.work.from.assigning.users.to.projects_v2.docx)
 updated from U.

**Comment by Forket** [comment]
This initial issue done.
Please check. Now assignments works.

**Comment by heinvv** [comment]
When user receives approval email, can you write this text:

Dear sir or madam,

(name surname) has added you as a developer to the following website listing on our website:
URL:
Role:

Accept button     Reject button


When user clicks on button, login, check credits, and afterwards give the message:

Congratulations, (name surname)! You have been added successfully to (URL) as a (role).

Or:
(without logging in)
Hereby we confirm that you have not been added to (URL) as a (role).


-----Original Message-----

**Comment by heinvv** [comment]
Can you send me a screenshot /video, what happens when user needs to buy more credits? What screen does he get to see?

-----Original Message-----

**Comment by Forket** [comment]
https://youtu.be/82wRtVgp9yM
